### ISSUE TITLE
Remove  hr-between-versions.rst template

### ISSUE THREAD

**Issue by adiroiban** [twisted/towncrier#646]
I see that it was introduced in towncrier 19.9.0  in https://github.com/twisted/towncrier/pull/162

but there are no tests for it.

I think that is best to remove it to simplify our code.

If anyone is using it, they can copy it to they custom path.

It's hard to support multiple template formats.

The default MD and RST should be enough.

**Comment by adiroiban** [comment]
@hynek I see that the template was mentioned as it was added for `attrs` , but I see that `attrs` now used a custom MD template.

I think that we are good to remove the template.

**Comment by adiroiban** [comment]
Maybe also remove single-file-no-bullets.rst

There are tests for it. I did a search of GitHub and I can't find anyone else using it... I am aware that GitHub public repos are not the whole universe :)

https://github.com/search?q=towncrier%3Asingle-file-no-bullets&type=code


The goal is to simplify the maintance of the project, having to care about fewer templates

**Comment by hynek** [comment]
Agreed, you can add something like “if you need them, get them from the repo” with a link to a previous tag.

most likely, they don’t work anymore anyways.
